deploy/docker/Dockerfile

```
@@ -1,6 +1,7 @@
 FROM ruby:2.5.1
 
-RUN apt-get update && apt-get install -y nodejs \
+RUN apt-get update && curl -sL https://deb.nodesource.com/setup_10.x | bash - \
+&& apt-get install -y nodejs \
```

The `curl | bash` and `apt update` should be swapped 
```suggestion
RUN curl -sL https://deb.nodesource.com/setup_10.x | bash \
&& apt-get update \ 
&& apt-get install -y nodejs \
```